src/tools/rbd_mirror/ServiceDaemon.cc

```diff
@@ -184,6 +184,13 @@ void ServiceDaemon<I>::remove_callout(int64_t pool_id, uint64_t callout_id) {
   schedule_update_status();
 }
 
+std::ostream& operator<<(std::ostream& out, const AttributeValue& value) {
```

Comment:
This is compensating for `std::variant` lacking https://www.boost.org/doc/libs/1_88_0/doc/html/boost/operator_1_3_43_5_4_1_3.html, right?

Reply:
correct.